src/BulkManager/BulkAction/EditHandler.php

```diff
@@ -129,15 +129,15 @@ public function bulkEditForm()
             FormAction::create('doSave', _t('GRIDFIELD_BULKMANAGER_EDIT_HANDLER.SAVE_BTN_LABEL', 'Save all'))
                 ->setAttribute('id', 'bulkEditingSaveBtn')
                 ->addExtraClass('btn btn-success')
-                ->setAttribute('data-icon', 'accept')
+                // ->setAttribute('data-icon', 'accept')
```

Comment:
Please don't comment out code. If it shouldn't be there, remove it entirely.

Reply:
i commented them because maybe it's possible to find the right attribute to set the proper icons but currently it's not working great as you can see in my screenshot :)

Reply:
If you have the time to find out if there's an appropriate attribute and set it, please do it in this PR - otherwise please remove the commented-out code.